chainreg/chainregistry.go

```diff
@@ -256,46 +219,22 @@ func GenDefaultBtcConstraints() channeldb.ChannelConstraints {
 //
 //nolint:lll
 func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
-	// Set the RPC config from the "home" chain. Multi-chain isn't yet
-	// active, so we'll restrict usage to a particular chain for now.
 	homeChainConfig := cfg.Bitcoin
-	if cfg.PrimaryChain() == LitecoinChain {
-		homeChainConfig = cfg.Litecoin
-	}
-	log.Infof("Primary chain is set to: %v", cfg.PrimaryChain())
-
 	cc := &PartialChainControl{
 		Cfg: cfg,
 	}
 
-	switch cfg.PrimaryChain() {
-	case BitcoinChain:
-		cc.RoutingPolicy = htlcswitch.ForwardingPolicy{
-			MinHTLCOut:    cfg.Bitcoin.MinHTLCOut,
-			BaseFee:       cfg.Bitcoin.BaseFee,
-			FeeRate:       cfg.Bitcoin.FeeRate,
-			TimeLockDelta: cfg.Bitcoin.TimeLockDelta,
-		}
-		cc.MinHtlcIn = cfg.Bitcoin.MinHTLCIn
-		cc.FeeEstimator = chainfee.NewStaticEstimator(
-			DefaultBitcoinStaticFeePerKW,
-			DefaultBitcoinStaticMinRelayFeeRate,
-		)
-	case LitecoinChain:
-		cc.RoutingPolicy = htlcswitch.ForwardingPolicy{
-			MinHTLCOut:    cfg.Litecoin.MinHTLCOut,
-			BaseFee:       cfg.Litecoin.BaseFee,
-			FeeRate:       cfg.Litecoin.FeeRate,
-			TimeLockDelta: cfg.Litecoin.TimeLockDelta,
-		}
-		cc.MinHtlcIn = cfg.Litecoin.MinHTLCIn
-		cc.FeeEstimator = chainfee.NewStaticEstimator(
-			DefaultLitecoinStaticFeePerKW, 0,
-		)
-	default:
-		return nil, nil, fmt.Errorf("default routing policy for chain "+
-			"%v is unknown", cfg.PrimaryChain())
+	cc.RoutingPolicy = htlcswitch.ForwardingPolicy{
+		MinHTLCOut:    cfg.Bitcoin.MinHTLCOut,
+		BaseFee:       cfg.Bitcoin.BaseFee,
+		FeeRate:       cfg.Bitcoin.FeeRate,
+		TimeLockDelta: cfg.Bitcoin.TimeLockDelta,
 	}
+	cc.MinHtlcIn = cfg.Bitcoin.MinHTLCIn
+	cc.FeeEstimator = chainfee.NewStaticEstimator(
+		DefaultBitcoinStaticFeePerKW,
+		DefaultBitcoinStaticMinRelayFeeRate,
+	)
```

Comment:
We could simplify by folding this all into one statement:
```go
cc := &PartialChainControl{
  Cfg: cfg,
  RoutingPolicy = htlcswitch.ForwardingPolicy{
    MinHTLCOut:    cfg.Bitcoin.MinHTLCOut,
    BaseFee:       cfg.Bitcoin.BaseFee,
    FeeRate:       cfg.Bitcoin.FeeRate,
    TimeLockDelta: cfg.Bitcoin.TimeLockDelta,
  },
  MinHtlcIn = cfg.Bitcoin.MinHTLCIn,
  FeeEstimator = chainfee.NewStaticEstimator(
    DefaultBitcoinStaticFeePerKW,
    DefaultBitcoinStaticMinRelayFeeRate,
  ),
}
```

We can also move the `cc` definition closer to the switch where it gets used.

Reply:
nice - done 👍